js/about/preferences.js

```diff
@@ -686,6 +686,21 @@ class PaymentsTab extends ImmutableComponent {
     </div>
   }
 
+  get nextReconcileDate () {
+    // Show date only after the user has funded the wallet.
+    // Afterwards balance should always !== 0 due to exchange rate fluctuations.
+    const ledgerData = this.props.ledgerData
+    const balance = Number(ledgerData.get('balance') || 0)
+    if (balance === 0 || !ledgerData.get('reconcileStamp')) {
```

Comment:
actually i think this should show up even if the balance is 0. there is sometimes a long delay between when the wallet is funded and when it actually shows up as balance.